### PR TITLE
add initial config

### PR DESCRIPTION
**Pull request by mkrystkowicz**

### CODE REVIEW

File: webpack.config.base.cjs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```
Comment (IraShuliar) [inline webpack.config.base.cjs:1]: I would move this file to webpack package, to have everything related to webpack in one place

File: src/views/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react"
```
Comment (IraShuliar) [inline src/views/Login/Login.js:1]: I would use also TypeScript in the project

File: src/styles/variables.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+$primary: #5E60CE;
```
Comment (IraShuliar) [inline src/styles/variables.scss:1]: Why you choose to not use styled components ?

**Comment by mkrystkowicz** [comment]
Closing PR, due to second approach choice - splitted for two ports.

File: src/styles/variables.scss (hunk 3 above)
Comment (mkrystkowicz) [reply src/styles/variables.scss:1]: Since it is start of the project, I can switch to styled components and typescript.

**Pull request closed without merging**